lib/basedriver/driver.js

```diff
@@ -62,6 +61,17 @@ class BaseDriver extends MobileJsonWireProtocol {
     this._eventHistory = {
       commands: [] // commands get a special place
     };
+
+    this.initLogger();
+  }
+
+  initLogger () {
+    const instanceId = Math.random().toString(16).substring(2, 8);
```

Comment:
sessionId is already a UUID, why do we care about an instance ID in addition?

Reply:
because session id is only created after we have a session and instance id exists when driver instance is constructed/session is deleted.